docs/LOGGING.md

```diff
@@ -30,28 +30,49 @@ Logs coming from xen-tools container are of three kinds:
   * ```qemu-dm-[VM_NAME]``` logs the qemu device model output
   * ```qdisk-[VM ID]``` logs the qdisk output
 
-If there are too many logs coming from various containers and the log reader is not able to keep up, oldest logs from the current circular buffer will be overwritten. All logs read by memlgod by rsyslogd will be written to disk queues. When there is heavy disk usage and rsyslogd does not get sufficient time to write incoming logs to disk, it can result in log loss from memlogd.
+All logs from memlgod and from /dev/kmsg read by newlogd will be written to disk log file and then to be compressed into gzip log files. When there is heavy disk usage or CPU load and newlogd does not get sufficient time to write incoming logs to disk, it can result in log loss from memlogd.
 
-The following diagram shows the flow of logs from containers to rsyslog and to cloud.
+The following diagram shows the flow of logs from containers to newlogd and to cloud.
 
-![EVE Log Flow](pics/eve_log_flow.png)
+![EVE Log Flow](pics/eve_newlog_flow.png)
 
-## Log Aggregation
+## Log Aggregation, Reformating and Compression for Persistent Log Files
 
-All logs collected from various containers/services in the system will reach rsyslogd demon. Rsyslogd queues the incoming logs on disk. Rsyslogd is configured to use disk queues in order to minimize log loss during power/network failures.
+All logs collected from various containers/services/kernel in the system will reach newlogd deamon. Newlogd formats the log entries and writes into temporary log files on disk in /persist/newlog/collect directory. The logs from device Dom0 side will be saved to file with name prefix with 'dev_log.' and logs from DomU side will be saved to files with name prefix with 'app.APP-UUID.log' where APP-UUID is the application UUID assigned to DomU application. The temporary file is kept on the disk until either the file size has exceeded 400 KBytes or the elasped time on the file has been opened for longer than 5 minutes.
```

Comment:
Instead of dom0/dumU can we use terminology which is more neutral such as "host" and "application/guest"? Guest tends to imply VM so application would be more neutral.

Reply:
Repeated "with"

Reply:
good point, will modify the unfriendly Doms :-)